source/setup/setup.py

```diff
@@ -85,28 +88,43 @@ def setup_bloodflow_model(self, PARAMETERS):
-                imp_transmiss = transmissibility.TransmissibilityVitroPries1992(PARAMETERS)
+                imp_transmiss = transmissibility.TransmissibilityVitroPries(PARAMETERS)
                 imp_velocity = rbc_velocity.RbcVelocityFahraeus(PARAMETERS)
             case 3:  # Takes RBCs into account based on the empirical laws by Pries and Secomb (2005)
                 imp_hd = discharge_haematocrit.DischargeHaematocritVitroPries2005(PARAMETERS)
-                imp_transmiss = transmissibility.TransmissibilityVitroPries2005(PARAMETERS)
+                imp_transmiss = transmissibility.TransmissibilityVitroPries(PARAMETERS)
                 imp_velocity = rbc_velocity.RbcVelocityFahraeus(PARAMETERS)
+            case 4:  # Discharging Hematocrit based on the empirical laws by Pries (1990)
+                imp_hd = discharge_haematocrit.DischargeHaematocritPries1990(PARAMETERS)
+                imp_transmiss = transmissibility.TransmissibilityVitroPries(PARAMETERS)
+                imp_velocity = rbc_velocity.RbcVelocityBulk(PARAMETERS)  # No Fahraeus effect (u_RBC = u_Bulk)
             case _:
                 sys.exit("Error: Choose valid option for the handling of RBCs (rbc_impact_option)")
 
         # Initialise the classes handling the solution of the linear system (build system and solver)
         match PARAMETERS["solver_option"]:
             case 1:
-                imp_buildsystem = build_system.BuildSystemSparseCoo(PARAMETERS)  # Fast approach to build the system
+                imp_buildsystem = build_system.BuildSystemSparseCsc(PARAMETERS)  # Fast approach to build the system
                 imp_solver = pressure_flow_solver.PressureFlowSolverSparseDirect(PARAMETERS)  # Direct solver
             case 2:
-                imp_buildsystem = build_system.BuildSystemSparseCoo(PARAMETERS)  # Fast approach to build the system
+                imp_buildsystem = build_system.BuildSystemSparseCsc(PARAMETERS)  # Fast approach to build the system
                 imp_solver = pressure_flow_solver.PressureFlowSolverPyAMG(PARAMETERS)  # Iterative solver
             case _:
                 sys.exit("Error: Choose valid option for the solver (solver_option)")
 
-        return imp_read, imp_write, imp_ht, imp_hd, imp_transmiss, imp_velocity, imp_buildsystem, imp_solver
+        match PARAMETERS["iterative_routine"]:
+            case 1:
+                imp_iterative = iterative_routine.IterativeRoutineNone(PARAMETERS)  # No iterative procedure
+            case 2 | 3 | 4:
+                imp_iterative = iterative_routine.IterativeRoutineMultipleIteration(PARAMETERS)
+            case _:
+                print("No Iterative Method selected: Default Selection")
+                imp_iterative = iterative_routine.IterativeRoutineNone(PARAMETERS)  # No iterative procedure
+
+        # Flow Balance
+        imp_balance = flow_balance.FlowBalanceClass(PARAMETERS)
```

Comment:
In that way, we don't have option for the flow mass balance, right? 
Can we discuss this point in person?